habitat/utils/visualizations/utils.py

```diff
@@ -164,22 +164,21 @@ def observations_to_image(observation: Dict, info: Dict) -> np.ndarray:
         generated image of a single frame.
     """
     egocentric_view_l: List[np.ndarray] = []
```

Comment:
Looks like it's not egocentric and pretty long non-descriptive name. Maybe rename to `visuals`, `obs_images`, `obs_displays` etc.